pyiron/project.py

```diff
@@ -113,6 +114,11 @@ def __init__(self, path="", user=None, sql_query=None, default_working_directory
         )
         self.job_type = JobTypeChoice()
         self.object_type = ObjectTypeChoice()
+        self._create = _Creator()
+
+    @property
+    def create(self):
+        return self._create
```

Comment:
```suggestion
```
These lines are no longer necessary when merging with the current master branch, as the property is already implemented in `pyiron_base`

Reply:
Yes, now that it's on pyiron base I will use that.